Gruntfile.js

```diff
@@ -19,11 +19,14 @@ function process( code, filepath ) {
 		.replace( /@DATE/g, ( new Date() ).toISOString().replace( /:\d+\.\d+Z$/, "Z" ) );
 }
 
+// Prevents CRLF EOL on Windows environments
```

Comment:
I included this second line: https://github.com/jquery/qunit/pull/941/files#diff-35b4a816e0441e6a375cd925af50752cR23